### DISCUSSION TITLE
Exclude subfolders from content folder in small navbar

### DISCUSSION THREAD

**Discussion by JayanthR19** [imfing/hextra#240] [Q&A]
Hi Team,

we have few subfolders in _content_ folder that we don't want to display in navbar. For that we removed the entries from _hugo.yaml_ menu section. After this folders are not getting displayed in big screens but when we are opening the site in a small screen the folders are getting listed in navbar.Please let us know do we have any options to address this.

**Comment by imfing** [comment]
@JayanthR19 Hi, there's no option to exclude entries from the navbar. 

I'll add support for this functionality in the future.

**Reply by JayanthR19** [reply]
@imfing thank you

**Comment by AntoninPvr** [comment]
@JayanthR19 Hi, I ran into the same issue. Here’s a small workaround that might help you. https://github.com/imfing/hextra/pull/883
